gobblin-data-management/src/main/java/org/apache/gobblin/util/commit/SetPermissionCommitStep.java

```diff
@@ -68,15 +67,9 @@ public void execute() throws IOException {
     for (Map.Entry<String, OwnerAndPermission> entry : pathAndPermissions.entrySet()) {
       Path path = new Path(entry.getKey());
       try {
-        if (!fs.exists(path)) {
-          log.info("Creating path {} with permission {}", path, entry.getValue().getFsPermission());
-          fs.mkdirs(path, entry.getValue().getFsPermission());
-          // Set owner and group since created directories will have owner set to the job runner (instead of the data owner)
-          fs.setOwner(path, entry.getValue().getOwner(), entry.getValue().getGroup());
-        } else {
-          log.info("Setting permission {} on existing path {}", entry.getValue().getFsPermission(), path);
-          fs.setPermission(path, entry.getValue().getFsPermission());
-        }
+        log.info("Setting permission {} on path {}", entry.getValue().getFsPermission(), path);
+        fs.setPermission(path, entry.getValue().getFsPermission());
+        // TODO : we can also set owner and group here.
```

Comment:
is this really a TODO?  looks like you just removed the impl. already present.  why?